<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWihEyes</class>
 <widget class="QMainWindow" name="EditWihEyes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>OnxFire</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1211</width>
      <height>802</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(22, 0, 66, 255), stop:1 rgba(0, 0, 0, 255));
color: rgb(89, 162, 170);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>50</y>
       <width>500</width>
       <height>500</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 0%);
border: 3px solid #347f74;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>C:/Users/jaazi/Downloads/logo2 (2).png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>50</y>
       <width>121</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
border: 3px solid #347f74;
background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(2, 85, 35, 255), stop:1 rgba(66, 64, 76, 255));</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QDial" name="dial">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(44, 94, 115);</string>
      </property>
     </widget>
     <widget class="QLabel" name="txt_brillo">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>0</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0,0%);
border:none;</string>
      </property>
      <property name="text">
       <string>Brillo</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>150</y>
        <width>70</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0,0%);
background-color: rgb(0, 0, 0);
border-color: rgb(85, 170, 255) ;
birder: 10px solid;
</string>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>250</y>
       <width>121</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:10px;
border: 3px solid #347f74;
background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(2, 85, 35, 255), stop:1 rgba(66, 64, 76, 255));</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="txt_brillo_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0,0%);
border:none;</string>
      </property>
      <property name="text">
       <string>Desenfoque</string>
      </property>
     </widget>
     <widget class="QDial" name="dial_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(44, 94, 115);</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber_2">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>150</y>
        <width>70</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0,0%);
background-color: rgb(0, 0, 0);
border-color: rgb(85, 170, 255) ;
birder: 10px solid;</string>
      </property>
      <property name="intValue" stdset="0">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn_OpenFile">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>131</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  
	background-color: qlineargradient(spread:pad, x1:0, y1:0.335, x2:1, y2:0, stop:0.432836 rgba(0, 92, 114, 255), stop:1 rgba(255, 255, 255, 255));
	border-radius:10px

}

QPushButton:hover {
	
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.335, x2:1, y2:0, stop:0.432836 rgba(114, 0, 104, 255), stop:1 rgba(255, 255, 255, 255));
	border-radius:10px
}</string>
     </property>
     <property name="text">
      <string>Abrir ...    </string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>C:/Users/jaazi/Downloads/62919cardindexdividers_109285.png</normaloff>C:/Users/jaazi/Downloads/62919cardindexdividers_109285.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>131</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  	border: none;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.335, x2:1, y2:0, stop:0.432836 rgba(0, 92, 114, 255), stop:1 rgba(255, 255, 255, 255));
	border-radius:10px

}

QPushButton:hover {
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.335, x2:1, y2:0, stop:0.432836 rgba(114, 0, 104, 255), stop:1 rgba(255, 255, 255, 255));
	border-radius:10px
}</string>
     </property>
     <property name="text">
      <string>Operacion</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>C:/Users/jaazi/Downloads/Edit_Image_icon-icons.com_54184.png</normaloff>C:/Users/jaazi/Downloads/Edit_Image_icon-icons.com_54184.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>50</y>
       <width>131</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  	border: none;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.335, x2:1, y2:0, stop:0.432836 rgba(0, 92, 114, 255), stop:1 rgba(255, 255, 255, 255));
	border-radius:10px

}

QPushButton:hover {
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.335, x2:1, y2:0, stop:0.432836 rgba(114, 0, 104, 255), stop:1 rgba(255, 255, 255, 255));
	border-radius:10px
}</string>
     </property>
     <property name="text">
      <string>Guardar...</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>C:/Users/jaazi/Downloads/save_14949.png</normaloff>C:/Users/jaazi/Downloads/save_14949.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>120</y>
       <width>201</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
border: 3px solid #347f74;
background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(2, 85, 35, 255), stop:1 rgba(66, 64, 76, 255));</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QRadioButton" name="rbt_Clahe">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Algoritmo Clahe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>65</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
background-color: rgba(0, 0, 0,0%);</string>
      </property>
      <property name="text">
       <string>Gridzise_1:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>95</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
background-color: rgba(0, 0, 0,0%);</string>
      </property>
      <property name="text">
       <string>Gridzise_2:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>125</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
background-color: rgba(0, 0, 0,0%);</string>
      </property>
      <property name="text">
       <string>Clip limit:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_Gridzisex">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>65</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_Gridzisey">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ln_Cliplimit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>120</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_6">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>320</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
border: 3px solid #347f74;
background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(2, 85, 35, 255), stop:1 rgba(66, 64, 76, 255));</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QRadioButton" name="rbt_Equ">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0, 0, 0,0%);
border:none;</string>
      </property>
      <property name="text">
       <string>Ecualizar</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>211</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
border: 3px solid #347f74;
background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(2, 85, 35, 255), stop:1 rgba(66, 64, 76, 255));</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Binarización</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="rbt_MedSimple">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Media Simple</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="rbt_GaussMed">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Media Gaussianna</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
